src/profile.c

```diff
@@ -764,9 +786,10 @@ Obj FuncACTIVATE_COLOR_PROFILING(Obj self, Obj arg)
 **
 *V  GVarFuncs . . . . . . . . . . . . . . . . . . list of functions to export
 */
-static StructGVarFunc GVarFuncs [] = {
+static StructGVarFunc GVarFuncs[] = {
 
-    GVAR_FUNC(ACTIVATE_PROFILING, 4, "string,boolean,boolean,integer"),
+    GVAR_FUNC(
+        ACTIVATE_PROFILING, 5, "string,boolean,boolean,boolean,integer"),
```

Comment:
Why isn't this "filename,coverage,wallTime,recordMem,resolution"? Using the argument names is the convention in all other files, at least.